### PR TITLE
Add Get-JiraIssueWorklog

### LINKED ISSUE #402: Add Get-JiraIssueWorklog
## Context

I'd like to be able to get a full list of issue worklogs. `Get-JiraIssue` only returns the first 20.

## Description

I'd like a function `Get-JiraIssueWorklog`, modeled after `Get-JiraIssueComment` that returns a full list of all worklog items for an issue. 

## Additional Information

Testing locally, I was able to accomplish this by copying the body of `Get-JiraIssueComment` to a new function `Get-JiraIssueWorklog`, changing only the URI (`comment` -> `worklog`) and the OutputType (`JiraComment` -> `JiraWorklogItem`). I also had to add `JiraWorklogItem` to the `ValidateSet` for `OutputType` on `Invoke-JiraMethod`.

I could submit a PR for this, but I'm not sure I'm able to write the appropriate unit tests.

### PR DESCRIPTION
**Pull request by asherber**
### Description

Add a new method, `Get-JiraIssueWorklog`, patterned after `Get-JiraIssueComment`

### Motivation and Context

Closes #402 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] My code follows the code style of this project.
- [x] I have added Pester Tests that describe what my changes should do.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by asherber** [comment]
I was unable to run the unit tests on my machine -- I think my environment is missing some configuration. And the automated tests seem to have been cancelled because the runner is unavailable.

File: docs/en-US/commands/Get-JiraIssueWorklog.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+external help file: JiraPS-help.xml
+Module Name: JiraPS
+online version: https://atlassianps.org/docs/JiraPS/commands/**Get-JiraIssueWorklog**/
```
Comment (lipkau) [inline docs/en-US/commands/Get-JiraIssueWorklog.md:4]: remote the `*` around the name of the function

**Review by lipkau — CHANGES_REQUESTED** [review]
please fix the 1 problem mentioned above.

The newly fixed CI shows that as the only problem

File: docs/en-US/commands/Get-JiraIssueWorklog.md (hunk 1 above)
Comment (asherber) [reply docs/en-US/commands/Get-JiraIssueWorklog.md]: Done, sorry about that!

**Review by lipkau — APPROVED** [review]

**Pull request merged**